CRM/Case/ActionMapping.php

```diff
@@ -84,6 +84,17 @@ public function getRecipientTypes() {
     return array('case_roles' => 'Case Roles');
   }
 
+  /**
+   * @inheritdoc
+   */
+  public function getDateFields() {
+    return [
+      'start_date' => ts('Case Start Date'),
+      'end_date' => ts('Case End Date'),
+      'case_status' => ts('Case Status Change'),
```

Comment:
Case status is not a date field, how come it is returned alongside other date fields?

Reply:
case_status is not directly a date field but it is. It is the date on which that case status was updated. 
So we dont have a direct way currently to find this date (its not stored).
Jamie said we would need to have discussions with core team to maybe add a new field or improve how case status changes are tracked.
But for now (before my last day), I was asked to see if there is a way to support this and using case activity was the only way I could do this.
I have explained this in the ticket CATL-1114.